src/sicmutils/generic.cljc

```diff
@@ -44,8 +40,7 @@
 
 (defn numerical-quantity?
   [x]
-  (or (number? x)
-      (abstract-number? x)
+  (or (abstract-number? x)
```

Comment:
`number?` is not our friend here, AND it's covered by `v/numerical?` below, which will return true for numbers now, always.